library/core/src/ptr/const_ptr.rs

```diff
@@ -520,9 +520,15 @@ impl<T: ?Sized> *const T {
     /// Calculates the distance between two pointers. The returned value is in
     /// units of T: the distance in bytes is divided by `mem::size_of::<T>()`.
     ///
-    /// This function is the inverse of [`offset`].
+    /// This is equivalent to `(self as isize - other as isize) / mem::size_of::<T>()`.
     ///
-    /// [`offset`]: #method.offset
+    /// The primary motivation of this method is for computing the `len` of an array/slice
+    /// of `T` that you are currently representing it as a "start" and "end" pointer
```

Comment:
```suggestion
    /// of `T` that you are currently representing as a "start" and "end" pointer
```
I think that's more grammatical?